src/pip/_internal/operations/install/wheel.py

```diff
@@ -84,14 +83,6 @@ def rehash(path, blocksize=1 << 20):
     return (digest, str(length))
 
 
-def csv_io_kwargs(mode):
-    # type: (str) -> Dict[str, Any]
-    """Return keyword arguments to properly open a CSV file
-    in the given mode.
-    """
-    return {'mode': mode, 'newline': '', 'encoding': 'utf-8'}
```

Comment:
Oh wait, the `newline` part is actually relevant, so I was wrong and we should keep this function… (not sure about the `encoding` part, but setting it explicitly is never a bad thing)

Reply:
That makes sense and thanks for debugging that.

In the latest revision:

- Restore `csv_io_kwargs`
- Rebase and resolve merge conflicts